makefile

```diff
@@ -40,7 +39,7 @@ test: debug
 	cd tests && ./test.d
 
 bin/dfmt-test: bin/githash.txt $(SRC)
-	$(DC) $(DMD_TEST_FLAGS) $^ -of$@
+	$(DC) $(DMD_TEST_FLAGS) $(filter %.d,$^) -of$@
```

Comment:
Here `bin/githash.txt` is supplied as a dependency which will get passed to the D compiler when using `$^`, and the call to filter (also used in the bin/dfmt target) will make sure only .d files are passed.